src/kudu/consensus/raft_consensus.cc

```diff
@@ -3870,6 +3870,9 @@ Status RaftConsensus::SetCurrentTermBootstrap(int64_t new_term) {
   }
   cmeta_->set_current_term(new_term);
   CHECK_OK(cmeta_->Flush());
+  if (vote_logger_) {
+    vote_logger_->advanceEpoch(new_term);
```

Comment:
I recall you have a SetVoteLogger function somewhere too. I think we should sync the epoch to kudu's epoch there too.

Reply:
I think that part is too early.

SetLogger is called here: https://github.com/facebook/kuduraft/blob/1.8.raft/src/kudu/tserver/simple_tablet_manager.cc#L486

ConcensusMeta is created in line 493, term bootstrap is done ins 531.